cypress/support/pages/user.pageObject.js

```diff
@@ -0,0 +1,25 @@
+import PageObject from '../PageObject';
+
+class userPageObject extends PageObject {
+  visitUserPage(username) {
+    cy.visit(`#/@${username}/`);
+  }
+
+  get profileUsername() {
+    return cy.getByDataCy('profile-username');
+  }
+  
+  get profileBio() {
+    return cy.getByDataCy('profile-bio');
+  }
+  
+  get unfollowBtn() {
+    return cy.getByDataCy('profile-unfollow');
+  }
+
+  get followBtn() {
+    return cy.getByDataCy('profile-follow');
+  }
+}
+
+export default userPageObject;
```

Comment:
The exported class name should be capitalized. Change 'userPageObject' to 'UserPageObject'.